routers/biocommons_groups.py

```diff
@@ -108,11 +111,27 @@ def approve_group_access(
         raise HTTPException(
             status_code=HTTPStatus.NOT_FOUND,
             detail="No membership request found for this user"
-        )
+    )
     membership.approval_status = ApprovalStatusEnum.APPROVED
     membership.updated_by = approving_user_record
     membership.grant_auth0_role(auth0_client=auth0_client)
     membership.save(session=db_session, commit=True)
+    if (
+        settings
+        and settings.send_email
+        and background_tasks is not None
+        and email_service is not None
+        and membership.user is not None
```

Comment:
Again, I'm not sure why `membership.user` would be `None` (and if it was it seems like something would be wrong somewhere else